src/scenes/Projects/Update/UpdateProjectDialog.tsx

```diff
@@ -0,0 +1,89 @@
+import { Typography } from '@material-ui/core';
```

Comment:
Can you rename the file so it matches the component name?

Reply:
Done